app/javascript/components/time_table/timesheet/tags_list.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TagsList extends React.Component {
+  static propTypes = {
+    tags: PropTypes.array,
+    selectedTag: PropTypes.string,
+  }
+
+  onChangeTag(tag_key) {
+    if (tag_key === this.props.selectedTag) {
+      this.props.onChangeTag('dev');
+    } else {
+      this.props.onChangeTag(tag_key);
+    }
+  }
+
+  render() {
+    const { selectedTag } = this.props;
+
+    return (
+      <div className="visible" tabIndex="-1" style={{ display: 'block !important' }}>
+        { this.props.tags.map((tag) => {
+          if (tag.key === 'dev') {
+            return null;
+          }
+          return (
+            <button
+              className={tag.key === selectedTag ? 'btn btn-success' : 'btn btn-info'}
+              onClick={this.onChangeTag.bind(this, tag.key)}
+              name="tag-item"
+              type="button"
+              is-selected={tag.key === selectedTag ? 'true' : 'false'}
```

Comment:
Why is string needed here?

Reply:
changed to '1 : 0', DOM would not accept boolean values